recipe/meta.yaml

```diff
@@ -1,42 +1,53 @@
 {% set label = "'full'" %}
 {% set tests = "['pywt']" %}
-{% set name = "PyWavelets" %}
-{% set version = "1.4.1" %}
+{% set name = "pywavelets" %}
+{% set version = "1.6.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 6437af3ddf083118c26d8f97ab43b0724b956c9f958e9ea788659f6a2834ba93
+  sha256: ea027c70977122c5fc27b2510f0a0d9528f9c3df6ea3e4c577ca55fd00325a5b
 
 build:
-  number: 1
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  number: 0
+  skip: True  # [py<39]
+  script:
+    - mkdir builddir
+    - {{ PYTHON }} -m mesonbuild.mesonmain setup $MESON_ARGS builddir  # [not win]
+    - {{ PYTHON }} -m mesonbuild.mesonmain setup %MESON_ARGS% builddir  # [win]
```

Comment:
Not sure if this is the cause of the failure, but it's not doing the right thing. That's why there's a warning like this:
```
WARNING: Using a debug build type with MSVC or an MSVC-compatible compiler
when the Python interpreter is not also a debug build will almost
certainly result in a failed build. Prefer using a release build
type or a debug Python interpreter.
```
The provided `MESON_ARGS` on this line get dropped in the reconfigure step:
```
+ meson setup --reconfigure %SRC_DIR% %SRC_DIR%\builddir -Dbuildtype=release -Db_ndebug=if-release -Db_vscrt=md --native-file=%SRC_DIR%\builddir\meson-python-native-file.ini
```

The robust way is passing on `MESON_ARGS` to the `python -m build` call like this: https://github.com/conda-forge/scipy-feedstock/blob/39c367144d7519ad131c7fdb073bf0d2476a374c/recipe/build.sh#L12-L23

Reply:
Ah yes, okay that is a good workaround for now. I'm aware of the problem, and a fix based on PEP 739 is in progress. See https://discuss.python.org/t/pep-739-static-description-file-for-build-details-of-python-installations/44572/56 for context.

Reply:
Including a change below to fix the Windows warning observed